receiver/solacereceiver/factory_test.go

```diff
@@ -30,9 +30,11 @@ func TestCreateTracesReceiver(t *testing.T) {
 	require.NoError(t, err)
 	require.NoError(t, component.UnmarshalConfig(sub, cfg))
 
+	set := receivertest.NewNopCreateSettings()
+	set.ID = component.NewIDWithName("solace", "factory")
 	receiver, err := factory.CreateTracesReceiver(
 		context.Background(),
-		receivertest.NewNopCreateSettings(),
+		set,
```

Comment:
Why is this needed?

Reply:
solace receiver registers a view in opencensus when it starts, based off the name of the receiver. The test generated uses the default name "", and that coincides with existing tests. Unfortunately, opencensus will return an error if you try to register the same metrics twice, so the workaround I found was to change the existing test to use a name so it would not clash with the generated test.